Source/Core/Core/Analytics.cpp

```diff
@@ -139,7 +139,8 @@ void DolphinAnalytics::ReportGameStart()
 
 // Keep in sync with enum class GameQuirk definition.
 static const char* GAME_QUIRKS_NAMES[] = {
-    "icache-matters",  // ICACHE_MATTERS
+    "icache-matters",                // ICACHE_MATTERS
```

Comment:
Nitpick: could you put the comments above their string rather than next to it? Otherwise every change to this array will involve updating all other lines, which makes for longer diffs.

Reply:
Will do. Was just following the existing style. :P